lib/class-vip-request-block.php

```diff
@@ -128,15 +128,17 @@ public static function block_and_log( string $value, string $criteria ) {
 			newrelic_ignore_transaction();
 		}
 
+		if ( defined( 'WP_CLI' ) && constant( 'WP_CLI' ) ) {
+			// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log
+			error_log( sprintf( 'VIP Request Block: would block based on "%s" with value of "%s" but it\'s a CLI request', $criteria, $value ) );
```

Comment:
The error log seems unnecessary; doesn't really add much.

Reply:
@mjangda why not? my reasoning was that if someone does this unintentionally they wouldn't know unless they'd look at their logs, at which point they might correct their mistake.

Reply:
If we're in a context where the request block stuff shouldn't run, we should just bail early and do nothing. That's better than asking customers to remember to check the context when implementing:

```
// Annoying
if ( ! defined( 'WP_CLI' ) && ! defined( 'DOING_CRON' ) ) {
    VIP_Request_Block::block( ... );
}

// Much better
VIP_Request_Block::block( ... );
```